paper-toolbar.html

```diff
@@ -102,11 +104,11 @@
     }
 
     :host(.medium-tall) {
-      height: 128px;
+      height: var(--paper-toolbar-height, 128px);
```

Comment:
@samccone what about if we use `calc(var(--paper-toolbar-height, 64px) * 2);`  and `calc(var(--paper-toolbar-height, 64px) * 3);` for `.tall`  Alternatively, we could have different custom properties for those classes.  wdyt? See this request: https://github.com/PolymerElements/paper-toolbar/issues/6